posts/2021/08/low-code-workshop.md

```diff
@@ -0,0 +1,120 @@
+These changes are made using GitHub's suggest changes feature that allows authors to make single line changes to code; a workflow perfect for alt text.
+
+### Group review
+
+Over 40 minutes several contributors can make quite a few commits. 
+The project contributor shares their screen to review the suggestions made over the course of the event. 
+This portion becomes a critique to review the different contributions made.
+
+### Bringing everything home
+
+All of the work we talked about happened on a contributor's fork. 
+They now have a pull request withl a gaggle of contributors. 
+Their last job is to submit the pull request to the mainline project and see the work accepted into the code base.
+
+## Success stories
+
+Everyone loves an underdog story. 
+What follows are there triumphs of some open source allies who thought they could run an hour sprint that allowed multiple folks to contribute to an open source project. 
+Believe it or not, we weren't lying to ourselves and others. Below we highlight our successes with running low-code sprints in Jupyter and NumPy.
+
+### Jupyter accessibility group
+
+There is a small group of the Jupyter community focused on the improving the accessibility of JupyterLab and orbiting projects. 
+We meet every other work to add small patches and fixes. Currently, we're in the process of organizing workshops to advocate for accessibility and improve the quality of the entire Jupyter experience for disabled scientists.
+
+On an off week, between our normal syncs, we tested a low-code format for folks to participate in the Jupyter project. 
+During an hour, four people were included in the project by suggesting changes to alt text for images in the JupyterLab documentation (https://github.com/isabela-pf/jupyterlab/pull/1), 
+resulting in a collaborative commit to mainline with the help of @krassowski and @isabela-pf as champions for the shared work.(https://github.com/jupyterlab/jupyterlab/pull/10670)
+
+We consider this event a success because we were able to remove `git` as a technical barrier to entry and support changes from multiple authors.
+
+
+### [NumPy documentation](https://numpy.org/doc/stable/) new comers meeting
+
+Our fledgling experience in the Jupyter event made us *sort of* confident this could work for other projects. 
+So we contacted our NumPy accessibility allies @melissawm and @marsbarlee to test this
+hypothesis at the NumPy newcomer's meeting, which [happens every other
+Thursday](http://numpy-discussion.10968.n7.nabble.com/Newcomer-s-Hour-td49226.html).
+
+During the meeting, @marsbarlee provided a valuable [slideshow on how to write alt text for
+scientific diagrams](https://docs.google.com/presentation/d/150vhbpGrtAc3ALhrS1a07lhEKCgevAY3ITh-4eCndDk/edit?usp=sharing). 
+This involves looking at examples and looking at the image context.
+
+The progress from the NumPy newcomer's meeting can be seen in this issue https://github.com/melissawm/numpy/pull/27 .
+
```

Comment:
```suggestion
@melissawm led the second iteration of this event to complete more alt text in the main NumPy documentation as well as beginning to cover [NumPy tutorials](https://numpy.org/numpy-tutorials/) with a new group of people. Including NumPy tutorials, which are originally written as Jupyter notebooks, also brought in questions about handling image or other types of media descriptions for cell outputs the tutorial produced. Working across repositories and markup languages, the community gathered to create another chunk of progress in more accessible documentation across the project. 
```

Reply:
@MarsBarLee @melissawm i was wondering it y'all might want to add anything here. @isabela-pf thanks for this update

Reply:
```suggestion
Melissa led the second iteration of this event to complete more alt text in the main NumPy documentation as well as beginning to cover [NumPy tutorials](https://numpy.org/numpy-tutorials/) with a new group of people. By hosting a workshop with another project, we saw the similar yet different challenges NumPy faced compared to Jupyter,

By working on the NumPy tutorials, which are originally written as Jupyter notebooks, we faced new questions, such as how to handling rendered live charts, which are not in still images formats such as JPEG. We faced other types of media, such as cell input and outputs.

We faced a variety of markup languages, including reST, Markdown and HTML, each with it's own quirks and slight variation in implementing alt-text.

Working across repositories and markup languages, the community gathered once again to create more accessible documentation across the open-source space, one contribution at a time.
```

Reply:
I just added on to Isabela's suggestion, with some more detail. I'm also not sure if you want to keep @github-handle vs using person-name. I went with person-name here but you can change it.